start-preview.yml

```diff
@@ -16,6 +16,13 @@ jobs:
         with:
           python-version: "3.7.x"
 
+      - name: Exports Google credentials
+        id: create-json
+        uses: jsdaniell/create-json@1.1.2
+        with:
+          name: "credentials.json"
```

Comment:
Let's give it a more specific name here since we might need to also create credential files for other warehouses. 
Can we skip this step if the secret isn't defined ?

Reply:
Yes, we can skip using `if` https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution#overview

I didn't bother because nothing breaks if the secret doesn't exist, but I guess it is better to check first..